pkg/executor/analyze_worker.go

```diff
@@ -59,7 +59,7 @@ func (worker *analyzeSaveStatsWorker) run(ctx context.Context, statsHandle *hand
 		}
 		err := statsHandle.SaveAnalyzeResultToStorage(results, analyzeSnapshot, util.StatsMetaHistorySourceAnalyze)
 		if err != nil {
-			logutil.Logger(ctx).Error("save table stats to storage failed", zap.Error(err))
+			logutil.Logger(ctx).Warn("save table stats to storage failed", zap.Error(err))
```

Comment:
wrapped by the foreground analyze's error or the error log of `auto analyze failed`.